add info about domain-with-org-lb (#2492)
18F/cg-site: _docs/services/external-domain-service.md

status: "Production Ready"
---

This service provides three different plans allowing you to use custom domains for your apps running on cloud.gov.

All plans offer:

1. Custom domain support, so that your application can have your domain instead of the default `*.app.cloud.gov` domain.
1. HTTPS support via free TLS certificates with auto-renewal (using [Let's Encrypt](https://letsencrypt.org/)), so that user traffic is encrypted.

> The domain-with-cdn plan also provides Content Distribution Network (CDN) caching (using [AWS CloudFront](https://aws.amazon.com/cloudfront/)) for fast delivery of content to your users.

Reviewer: ```suggestion
The `domain-with-cdn` plan also provides Content Distribution Network (CDN) caching (using [AWS CloudFront](https://aws.amazon.com/cloudfront/)) for fast delivery of content to your users.
```

> The domain-with-org-lb plan is offers load balancers dedicated to your cf organization. This plan is not enabled by default - reach out if you'd like to enable it

Reviewer: ```suggestion
The domain-with-org-lb plan is offers load balancers dedicated to your Cloud Foundry organization. This plan is not enabled by default for all organizations, but please contact [**{{site.support_email_address}}**]({{site.support_email}}) if you are interested in using it.
```

> The domain-with-org-lb plan is offers load balancers dedicated to your Cloud Foundry organization. This plan is not enabled by default for all organizations, but please contact [**{{site.support_email_address}}**]({{site.support_email}}) if you are interested in using it.
> for your org.

Reviewer: ```suggestion

The `domain-with-org-lb` plan is offers load balancers dedicated to your Cloud Foundry organization. This plan is not enabled by default for all organizations, but please contact [**{{site.support_email_address}}**]({{site.support_email}}) if you are interested in using it.
for your org.
```

## Update an instance

Not all plans can be updated - see the matrix below for options.
If you'd like to make an update not supported by the broker, you must delete and re-create your service

<table style="text-align:center;">
        <tr>
            <td colspan=2 rowspan=2></td>
            <th style="text-align:center;" colspan=3 scope="colgroup">New Plan</th>
        </tr>
        <tr>
            <th scope="col">domain</th>
            <th scope="col">domain-with-cdn</th>
            <th scope="col">domain-with-org-lb</th>
        </tr>
        <tr>
            <th style="vertical-align:center" rowspan=5 scope="rowgroup">Existing plan</th>
        </tr>
        <tr>
            <th scope="row">domain</th>
            <td>✅</td>
            <td>🚫</td>
            <td>✅</td>
        </tr>
        <tr>
            <th scope="row">domain-with-cdn</th>
            <td>🚫</td>
            <td>✅</td>
            <td>🚫</td>
        </tr>
        <tr>
            <th scope="row">domain-with-org-lb</th>
            <td>🚫</td>
            <td>🚫</td>
            <td>🚫</td>
        </tr>

> </table>

Reviewer: Can we format this using Markdown table syntax for easier maintenance?

https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables

Author: I used an HTML table because Markdown doesn't support merged cells (afaict)

Reviewer: Table looks really nice in the Pages preview build

## Update an instance

Not all plans can be updated - see the matrix below for options.
If you'd like to make an update not supported by the broker, you must delete and re-create your service

<table style="text-align:center;">
        <tr>
            <th></th>
            <th style="text-align:center;" colspan=4>New Plan</th>
        </tr>
        <tr>
            <th colspan=2></th>
            <th scope="col">domain</th>
            <th scope="col">domain-with-cdn</th>
            <th scope="col">domain-with-org-lb</th>
        </tr>
        <tr>
            <td style="vertical-align:center" rowspan=5>Exising plan</td>
        </tr>
        <tr>
            <td>domain</td>
            <td>✅</td>
            <td>🚫</td>
            <td>✅</td>
        </tr>
        <tr>
            <td>domain-with-cdn</td>
            <td>🚫</td>
            <td>✅</td>
            <td>🚫</td>
        </tr>
        <tr>
            <td>domain-with-org-lb</td>
            <td>🚫</td>
            <td>🚫</td>
            <td>🚫</td>
        </tr>

> </table>

Author: ![image](https://github.com/cloud-gov/cg-site/assets/15840717/c69d46c3-647b-4bf8-8391-685aa1982557)
